src/components/WebexRemoteMedia/WebexRemoteMedia.scss

```diff
@@ -1,17 +1,21 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-remote-media {
   min-height: $WEBEX_MEETING_MIN_HEIGHT;
   min-width: $WEBEX_MEETING_MIN_WIDTH;
-  background: white;
+  background: var(--wxc-meeting-background);
 
   position: relative;
   display: flex;
   justify-content: center;
   align-items: center;
+  flex-direction: column;
 
-  .md-alert-banner {
-    position: absolute;
-    top: 0;
-    width: 100%;
+  .text-alert {
+    font-family: 'CiscoSansTT';
+    font-size: 0.875rem;
+    line-height: 1.3125rem;
+    color: var(--wxc-text-color);
+    margin-top: 1.234rem;
+    text-align: center;
```

Comment:
you can drop this and just add 2 divs for the alert message instead of using `<br/>`